test/react-component-features/properties-and-stylable.spec.tsx

```diff
@@ -1,5 +1,5 @@
 import {properties, stylable} from "../../src";
-import {createGenerator} from "stylable";
+import {createGenerator} from "stylable/dist/src/generator";
```

Comment:
No.....
We will only use Stylable's API. pulling it from /dist is a step backwards